src/components/Link/docs/Link.md

```diff
@@ -0,0 +1,22 @@
+The `Link` component is valuable mainly because it's hooked up to Themer.
```

Comment:
if this is the main value prop of this component, we should add a test or two to test that themer changes change the component output